### ISSUE TITLE
Support Android Studio

### ISSUE THREAD

**Issue by donniezazen** [owncloud/android#831]
Android Studio has seen its 1.0 release and is now defacto IDE for Android development. Are there any plans to support or switch to Android Studio anytime soon?

**Comment by tobiasKaminsky** [comment]
As we have gradle support in branch develop it should be very simple:
https://developer.android.com/sdk/installing/migrate.html
But I have not tried it.

**Comment by jabarros** [comment]
Hi @donniezazen 
As @tobiasKaminsky says, 'develop' branch is prepared for being used with Android Studio IDE. 
Please, follow the steps commented in SETUP.md file: https://github.com/owncloud/android/blob/develop/SETUP.md#6-building-with-android-studio

**Comment by tobiasKaminsky** [comment]
@donniezazen Have you tried it and I assume that it worked.
If not, please open this issue again and explain with details where the problem is.

**Comment by donniezazen** [comment]
@jabarros @tobiasKaminsky My apologies. I am traveling these days and mobile internet is not that great.

I did as instrusted in the [link](https://github.com/owncloud/android/blob/develop/SETUP.md#6-building-with-android-studio). It complained `failed to find Build Tools revision 20.0.0`  which I obviously don't have. The latest version is 21.1.2. I updated `build.gradle` `classpath 'com.android.tools.build:gradle:1.0.0'`, `compileSdkVersion 21` and `buildToolsVersion "21.1.2"` But it still wants to download 20.0.0 build tools. `dependencies` and `main`, in `build.gradle`, both complain `cannot be applied to (groovy.lang.closure)`

PS:- The main screen of Android Studio doesn't have Import Project. Import Project has to be accessed from other Projects. It has three relevant options 1. "Start a new AS project" 2. "Open a existing AS project" and 3. "Import Non-AS project." Also I suppose AS can build on its own and doesn't need go to command line, so, "To build the project, follow the guidelines shown on 4. Building with Gradle." is not really needed.
